src/python/setup.py

```diff
@@ -93,16 +107,31 @@ def deepgnn_tf(version: str):
         name="deepgnn-tf",
         version=version,
         description="DeepGNN algorithms for tensorflow.",
+        long_description="See [DeepGNN package](https://pypi.org/project/deepgnn-ge/) for detailed description.",
```

Comment:
Many repos make this == the README with file = open(filename) and file.readlines() if you are interested.

Reply:
That's what we do on line 40. Here we reference the main package.